UI/Filters/ExceptionLoggingMiddleware.cs

```diff
@@ -13,6 +13,16 @@ public async Task InvokeAsync(HttpContext context)
         try
         {
             await _next(context);
+
+            if (context.Response.StatusCode == 404)
+            {
+                using var _ = _logger.BeginScope("{@Context}", context);
+                var url = $"{context.Request.Scheme}://{context.Request.Host}{context.Request.Path}{context.Request.QueryString}";
+                var referer = context.Request.Headers["Referer"].ToString();
```

Comment:
Can we strip the `Referer` to the scheme and hostname (and strip off the path, query string, and fragment)?

Reply:
Have updated so just logging host